web/template/layout/mlpa-header.gohtml

```diff
@@ -5,41 +5,59 @@
-                <a class="govuk-tabs__tab" href="#history">
-                    History
-                </a>
-            </li>
+    <nav class="moj-sub-navigation" aria-label="linked cases">
+
+        <ul class="moj-sub-navigation__list">
+            {{ range .SortedLinkedCases }}
+                <li class="moj-sub-navigation__item">
+                    <a class="moj-sub-navigation__link" {{ if eq $.Lpa.UID .UID }}aria-current="page"{{ end }} href="{{ prefix (printf "/lpa/%s" .UID ) }}">
+                        {{ template "status-tag" (ToLower $.Lpa.Status) }} {{ ToUpper .Subtype }} {{ .UID }}
+                    </a>
+                </li>
+            {{ end }}
         </ul>
-    </div>
+
+        <h2 class="govuk-heading-m govuk-!-margin-top-3">
+            {{ template "status-tag" (ToLower $.Lpa.Status) }} {{ if (eq .Lpa.Subtype "hw") }}Health and welfare{{ else }}Property and finance{{ end }}, {{ .Lpa.UID }}
+        </h2>
+
+        <div class="govuk-tabs">
+            <h2 class="govuk-tabs__title">
+                Contents
+            </h2>
+            <ul class="govuk-tabs__list">
+                <li class="govuk-tabs__list-item {{ if eq .TabName "summary" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s" .Lpa.UID) }}">
+                        Summary
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "details" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="#lpa-details">
+                        LPA details
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "issues" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab"  href="#issues">
+                        Issues
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "fees" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s/payments" .Lpa.UID) }}">
+                        Fees
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "documents" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s/documents" .Lpa.UID) }}">
+                        Documents
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "history" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="#history">
+                        History
+                    </a>
+                </li>
+            </ul>
+        </div>
+
+    </nav>
```

Comment:
I think this should be before the tabs, otherwise it suggests to a screenreader that the tabs are also related to linked cases.